app/models/concerns/form_validations.rb

```diff
@@ -0,0 +1,13 @@
+require 'active_support/concern'
+
+module FormValidations extend ActiveSupport::Concern
```

Comment:
Can `extend ActiveSupport::Concern` be on a newline to fix the indentation of this file?